recipes/holoscan/meta.yaml

```diff
@@ -0,0 +1,389 @@
+      description: |
+        This is a runtime package only. Developers should install holoscan (python) to build with Holoscan
+
+  - name: holoscan
+    files:
+      - examples/
+      - lib/python{{ python_version }}/site-packages/gxf
+      - lib/python{{ python_version }}/site-packages/holoscan
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+        - {{ stdlib("c") }}
+        - arm-variant * {{ arm_variant_type }}  # [aarch64]
+      host:
+        - {{ pin_subpackage("libholoscan", exact=True) }}
+        - cuda-version {{ cuda_version }}
+        - python ={{ python_version }}
+      run:
+        - arm-variant * {{ arm_variant_type }}  # [aarch64]
+        - {{ pin_subpackage("libholoscan", exact=True) }}
+        - cuda-cudart
+        - python ={{ python_version }}
+    test:
+      imports:
+        - holoscan
+        - gxf
+      requires:
+        - pip
+      commands:
+        - test -d $PREFIX/lib/python{{ python_version }}/site-packages/gxf
+        - test -d $PREFIX/lib/python{{ python_version }}/site-packages/holoscan
+        - pip check
+        - python -c "import holoscan; print(holoscan.__version__)"
+    about:
+      license: LicenseRef-NVIDIA-End-User-License-Agreement
+      license_file: LICENSE
+      license_url: https://developer.download.nvidia.com/assets/Clara/NVIDIA_Clara_EULA.pdf
+      summary: NVIDIA Holoscan python bindings
+      description: |
+        NVIDIA Holoscan python bindings
+
+  - name: libholoscan-dev
+    build:
+      skip: true
+      missing_dso_whitelist:
+        - "*libgxf_app.so"                                   # Packaged in libholoscan lib, but referened by examples/** here
+        - "*libgxf_core.so"                                  # Packaged in libholoscan lib, but referened by examples/** here
+        - "*libgxf_holoscan_wrapper_lib.so"                  # Packaged in libholoscan lib, but referened by examples/** here
+        - "*libgxf_multimedia.so"                            # Packaged in libholoscan lib, but referened by examples/** here
+        - "*libgxf_rmm.so"                                   # Packaged in libholoscan lib, but referened by examples/** here
+        - "*libgxf_std.so"                                   # Packaged in libholoscan lib, but referened by examples/** here
+        - "*libgxf_sample.so"                                # Packaged in libholoscan lib, but referened by examples/** here
+        - "*libgxf_serialization.so"                         # Packaged in libholoscan lib, but referened by examples/** here
+        - "*libgxf_ucx.so"                                   # Packaged in libholoscan lib, but referened by examples/** here
+        - "*libholoscan_*.so.3"                              # Packaged in libholoscan lib, but referened by python/** here
+        - "*libyaml-cpp.so.0.7"                              # Packaged in libholoscan lib, but referened by python/** here
```

Comment:
If these libraries are available in libholoscan. Then why no just add libholoscan as a host dependency as suggested in the previous review comment?

https://github.com/conda-forge/staged-recipes/pull/28598#discussion_r1962190367

Reply:
Added the missing host dependency